solutions/gold/cses-1681.mdx

```diff
@@ -139,4 +144,51 @@ public class cses1681 {
 }
 ```
 </JavaSection>
+
+<PySection>
+
+```py
+from collections import deque
+
+MAX_N = 10**5 + 1
+MOD = 10**9 + 7
+
+edge = [[] for _ in range(MAX_N)]
+backedge = [[] for _ in range(MAX_N)]
```

Comment:
why not just initialize in terms of `n` instead

Reply:
done